sootty/storage/wiregroup.py

```diff
@@ -39,10 +37,29 @@ def find(self, name: str):
         raise SoottyError(f"Wire '{name}' does not exist.")
 
     def get_names(self):
-        """Returns list of all wire names."""
-        names = set()
-        for wire in self.wires:
-            names.add(wire.name)
-        for group in self.groups:
-            names.update(group.get_names())
+        """Returns a dictionary of all wire names of this wiregroup or a list if this wiregroup is the innermost one."""
```

Comment:
Can you remove the duplicate changes from PR #64? I approved so once you do that you can merge it in.

Reply:
Removed